.kitchen.dokken.yml

```diff
@@ -1,7 +1,7 @@
 ---
 driver:
   name: dokken
-  privileged: true
+  privileged: false
```

Comment:
can't we just remove this line? Isn't it default

Reply:
Just testing some stuff, to do with /sbin/init and agetty

Reply:
Seemingly the CentOS-6 builds are hitting this issue

https://github.com/moby/moby/issues/4040